modules/currency-l0/src/main/scala/org/tessellation/currency/l0/Main.scala

```diff
@@ -10,5 +10,6 @@ object Main
       "Currency-l0",
       "Currency L0 node",
       ClusterId(UUID.fromString("517c3a05-9219-471b-a54c-21b7d72f4ae5")),
-      version = BuildInfo.version
-    ) {}
+      tessellationVersion = BuildInfo.version,
+      metagraphVersion = "0"
```

Comment:
should it be `0` or just empty?

Reply:
Actually can be empty ""

Reply:
if it's going to be empty, maybe it's a case for the optional type? `Option[Version]`?

Reply:
It can be empty here on tessellation, but I want to ensure that everyone who implements those classes provides a version.